components/Layout/Header.tsx

```diff
@@ -0,0 +1,107 @@
+import Link from 'next/link';
+import { useState } from 'react';
+import { hoverClass, headerLinks } from 'styles/globals';
+
+export const Header = () => {
+  const [showMobileMenu, setShowMobileMenu] = useState(false);
+  const toggleMobileMenu = () => {
+    setShowMobileMenu(!showMobileMenu);
+  };
+
+  return (
+    <header className="bg-stone-200">
+      <div className="mx-auto max-w-screen-2xl px-4 sm:px-6 lg:px-8">
+        <div className="flex h-24 items-center justify-between">
+          <div className="md:flex md:items-center md:gap-12 mr-6 xl:mr-12 p-4 max-xl:p-2">
+            <Link className="block" href="/">
+              Valory logo
+            </Link>
+          </div>
+
+          <div className="hidden lg:block">
+            <nav aria-label="Global">
+              <ul className="flex items-center gap-6 xl:gap-12 text-sm">
+                {headerLinks.map((link, index) => (
+                  <li key={index} className={hoverClass}>
+                    {link.isExternal ? (
+                      <a className={hoverClass} href={link.url} target="_blank">
+                        {' '}
```

Comment:
why do we need {' '} before and after the link label? the same for internal links. maybe can remove?